core/validation/validate.py

```diff
@@ -68,12 +68,14 @@ def validate_workbook(workbook: dict[str, pd.DataFrame], schema: dict) -> list[v
         (validate_uniques, "uniques"),
         (validate_foreign_keys, "foreign_keys"),
         (validate_enums, "enums"),
-        (validate_nullable, "non-nullable"),
+        (validate_nullable, "non-nullable"),  # TODO: add tests for this
     )
 
     validation_failures = []
     for sheet_name in workbook.keys():
-        if workbook[sheet_name].empty:
+        # if the table is empty and not defined as nullable, then raise an Empty Sheet Failure
+        # TODO: add tests for this
```

Comment:
could we bang a quick test in while we're here, looks like it should be easy? Means we don't add that to the tech-debt mountain 😄

Reply:
yeah sure

Reply:
Added